<launch>
    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="log">
        <param name="video_device" value="/dev/video0" />
        <!-- <param name="image_width" value="640" />
        <param name="image_height" value="480" />
        <param name="pixel_format" value="yuyv" />
        <param name="io_method" value="mmap"/> -->

	<remap from="/usb_cam" to="/vision/front_cam"/>
    </node>
    <node name="parse_front_cam" pkg="vision" type="parse_front_cam.py" respawn="false" output="screen"/>
</launch>
